unknown pragma noise

Building Entwine against current master causes a lot of the following warning on clang 9.0.0 on OSX:

```
/Users/hobu/pdal-build/include/pdal/util/Uuid.hpp:65:9: warning: unknown pragma ignored [-Wunknown-pragmas]
#pragma warning(push)
        ^
/Users/hobu/pdal-build/include/pdal/util/Uuid.hpp:66:9: warning: unknown pragma ignored [-Wunknown-pragmas]
#pragma warning(disable: 4267 4244)  // ignore conversion warnings
        ^
/Users/hobu/pdal-build/include/pdal/util/Uuid.hpp:235:9: warning: unknown pragma ignored [-Wunknown-pragmas]
#pragma warning(pop)
```

--- a/pdal/PointView.hpp
+++ b/pdal/PointView.hpp
@@ -48,7 +48,7 @@
 #include <set>
 #include <deque>
 
-#pragma warning(disable: 4244)  // conversion from 'type1' to 'type2', possible loss of data
+//#pragma warning(disable: 4244)  // conversion from 'type1' to 'type2', possible loss of data
 
 namespace pdal
 {

--- a/pdal/util/Uuid.hpp
+++ b/pdal/util/Uuid.hpp
@@ -62,10 +62,6 @@
 #include "Inserter.hpp"
 #include "Extractor.hpp"
 
-#pragma warning(push)
-#pragma warning(disable: 4267 4244)  // ignore conversion warnings
-
-
 namespace pdal
 {
 
@@ -231,6 +227,4 @@ inline std::istream& operator >> (std::istream& in, Uuid& u)
 }
 
 } // namespace pdal
-
-#pragma warning(pop)
 

